robotoff/utils/sql/jsonl_to_parquet.sql

```diff
@@ -18,7 +18,7 @@ COPY (
         completeness,
         correctors_tags,
         countries_tags,
-        created_t,
+        to_timestamp(created_t)::datetime AS created_t, -- Convert from unixtime to datetime
```

Comment:
Hum I'm not sure to understand why you do this?
`created_t` is an int, and it looks it's correctly detected by Parquet.

Reply:
Indeed, but it is a Unix timestamp, which is not particularly obvious to someone new to the dataset

Reply:
> Indeed, but it is a Unix timestamp, which is not particularly obvious to someone new to the dataset

We should keep it as it, so that we keep as much as possible the compatibility with the other database dump (JSONL, MongoDB, API, etc).
We can add a `created_datetime` version, just like the CSV does, with the converted datetime though!